### DISCUSSION TITLE
Help - Copy the existing directus tabular layout and make a minor change and create it as a tabular2  layout in the extension

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#13059] [Archive]
Copy the existing directus tabular layout and make a minor change and create it as a tabular2  layout in the extension

Is it possible.?

I copied the source file of directus tabular layout and tried compiling without making any change initially but I get the following error

![image](https://user-images.githubusercontent.com/36895131/165929048-02019d24-c9a1-469e-ad00-e073f3a81a4e.png)

Can someone guide me with the steps pls

**Comment by rijkvanzanten** [comment]
The core layouts share a lot of utilities and other functions with other core parts of the app, that aren't all exposed to extensions. You can't just copy paste the whole layout and have it work at the moment 👍🏻
